src/new_to_cabal/01_installing_cabal_and_ghc.md

```diff
@@ -1 +1,73 @@
 # Installing cabal + GHC
+
+## ghcup
+
+While most package managers have some support for installing at least ghc (and
+sometimes cabal) the suggested method these days is `ghcup`. You can find
+official installation instructions
+[here](https://github.com/haskell/ghcup-hs#installation). You should be able to
+run the curl command from https://www.haskell.org/ghcup/. Once you have ghcup
```

Comment:
I think I'd just mention the second link, where a link to the first location is given, as well.